src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -228,18 +230,40 @@ public Table (String name, Class<? extends Entity> entityClass, Requirement requ
     public static final Table STOPS = new Table("stops", Stop.class, REQUIRED,
         new StringField("stop_id",  REQUIRED),
         new StringField("stop_code",  OPTIONAL),
-        new StringField("stop_name",  REQUIRED),
+        new StringField("stop_name",  OPTIONAL).requireConditions(),
         new StringField("stop_desc",  OPTIONAL),
-        new DoubleField("stop_lat", REQUIRED, -80, 80, 6),
-        new DoubleField("stop_lon", REQUIRED, -180, 180, 6),
-        new StringField("zone_id",  OPTIONAL),
+        new DoubleField("stop_lat", OPTIONAL, -80, 80, 6).requireConditions(),
+        new DoubleField("stop_lon", OPTIONAL, -180, 180, 6).requireConditions(),
+        new StringField("zone_id", OPTIONAL).foreignFieldReference(),
```

Comment:
Maybe rename to just `foreign()`?

Reply:
Updated. I think the whole naming terminology used in this PR may need to be looked at.